Installation/cmake/modules/FindSuiteSparse.cmake

```diff
@@ -0,0 +1,437 @@
+## CMake file to locate SuiteSparse and its useful composite projects
```

Comment:
The file seems to be a copy-paste of some exists cmake file that should be shipped by the library itself. So except if it is not shipped by the library or because you did some modifications, I don't see why we should ship it.